### ISSUE TITLE
[BUG] gpu-2.0.0-ls42 don't work

### ISSUE THREAD

**Issue by MorganMLGman** [linuxserver/docker-faster-whisper#22]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

```
whisper  | [migrations] started
whisper  | [migrations] no migrations found
whisper  | ───────────────────────────────────────
whisper  |
whisper  |       ██╗     ███████╗██╗ ██████╗
whisper  |       ██║     ██╔════╝██║██╔═══██╗
whisper  |       ██║     ███████╗██║██║   ██║
whisper  |       ██║     ╚════██║██║██║   ██║
whisper  |       ███████╗███████║██║╚██████╔╝
whisper  |       ╚══════╝╚══════╝╚═╝ ╚═════╝
whisper  |
whisper  |    Brought to you by linuxserver.io
whisper  | ───────────────────────────────────────
whisper  |
whisper  | To support LSIO projects visit:
whisper  | https://www.linuxserver.io/donate/
whisper  |
whisper  | ───────────────────────────────────────
whisper  | GID/UID
whisper  | ───────────────────────────────────────
whisper  |
whisper  | User UID:    1000
whisper  | User GID:    1000
whisper  | ───────────────────────────────────────
whisper  | Linuxserver.io version: 2.0.0-ls42
whisper  | Build-date: 2024-10-27T06:41:16+00:00
whisper  | ───────────────────────────────────────
whisper  |
whisper  | [custom-init] No custom files found, skipping...
whisper  | INFO:__main__:Ready
whisper  | Connection to localhost (127.0.0.1) 10300 port [tcp/*] succeeded!
whisper  | [ls.io-init] done.
whisper  | INFO:faster_whisper:Processing audio with duration 00:10.880
whisper  | Unable to load any of {libcudnn_ops.so.9.1.0, libcudnn_ops.so.9.1, libcudnn_ops.so.9, libcudnn_ops.so}
whisper  | Invalid handle. Cannot load symbol cudnnCreateTensorDescriptor
whisper  | INFO:__main__:Ready
```

### Expected Behavior

```
whisper  | [migrations] started
whisper  | [migrations] no migrations found
whisper  | ───────────────────────────────────────
whisper  |
whisper  |       ██╗     ███████╗██╗ ██████╗
whisper  |       ██║     ██╔════╝██║██╔═══██╗
whisper  |       ██║     ███████╗██║██║   ██║
whisper  |       ██║     ╚════██║██║██║   ██║
whisper  |       ███████╗███████║██║╚██████╔╝
whisper  |       ╚══════╝╚══════╝╚═╝ ╚═════╝
whisper  |
whisper  |    Brought to you by linuxserver.io
whisper  | ───────────────────────────────────────
whisper  |
whisper  | To support LSIO projects visit:
whisper  | https://www.linuxserver.io/donate/
whisper  |
whisper  | ───────────────────────────────────────
whisper  | GID/UID
whisper  | ───────────────────────────────────────
whisper  |
whisper  | User UID:    1000
whisper  | User GID:    1000
whisper  | ───────────────────────────────────────
whisper  | Linuxserver.io version: 2.0.0-ls41
whisper  | Build-date: 2024-10-20T06:40:45+00:00
whisper  | ───────────────────────────────────────
whisper  |
whisper  | [custom-init] No custom files found, skipping...
whisper  | INFO:__main__:Ready
whisper  | Connection to localhost (127.0.0.1) 10300 port [tcp/*] succeeded!
whisper  | [ls.io-init] done.
whisper  | INFO:faster_whisper:Processing audio with duration 00:15.000
whisper  | INFO:wyoming_faster_whisper.handler: Dziękuje za oglądanie.
whisper  | INFO:faster_whisper:Processing audio with duration 00:15.000
whisper  | INFO:wyoming_faster_whisper.handler: Dziękuje za oglądanie.
whisper  | INFO:faster_whisper:Processing audio with duration 00:02.080
whisper  | INFO:wyoming_faster_whisper.handler: Jaki jest dziś dzień
```

### Steps To Reproduce

update container

### Environment

```markdown
- OS: Ubuntu 24.04
- How docker service was installed: distro's packagemanager
```

### CPU architecture

x86-64

### Docker creation

```bash
whisper:
    image: lscr.io/linuxserver/faster-whisper:gpu-2.0.0-ls41
    container_name: whisper
    environment:
      - PUID=$PUID
      - PGID=$PGID
      - TZ=$TZ
      - WHISPER_MODEL=large-v3
      - WHISPER_BEAM=2 #optional
      - WHISPER_LANG=pl #optional
      - NVIDIA_VISIBLE_DEVICES=all
      - NVIDIA_DRIVER_CAPABILITIES=compute,video,utility
    deploy:
      resources:
        reservations:
          devices:
            - capabilities: [gpu]
    volumes:
      - $DOCKER_APPS_DIR/whisper:/config
    ports:
      - 10300:10300
    restart: unless-stopped
    labels:
      - "com.centurylinklabs.watchtower.enable=true"
```
```


### Container logs

```bash
already uploaded
```

**Comment by thespad** [comment]
Looks like ctranslate2 has updated to only support CUDNN 9 but faster-whisper only supports CUDNN 8 so we might have to pin it.

**Comment by thespad** [comment]
Pushing two builds to https://github.com/linuxserver/docker-faster-whisper/pull/23, image names will get posted by the bot once they're available.

First pins ctranslate2 to 4.4.0 with CUDNN 8
Second upgrades CUDNN to 9.

Various linked discussion from https://github.com/SYSTRAN/faster-whisper/issues/1086 suggests faster whisper should work with ctranslate 4.5.0 despite what the docs say, so that one would be preferable rather than pinning ctranslate2 to an older version.

Please can you test them and let me know if one/both/neither work for you.

**Comment by MorganMLGman** [comment]
`ghcr.io/linuxserver/lspipepr-faster-whisper:gpu-2.0.0-pkg-b2138dcc-dev-550713281c67d3de93e9724577979e887a40ccdc-pr-23`

Works

```
whisper  | [migrations] started
whisper  | [migrations] no migrations found
whisper  | ───────────────────────────────────────
whisper  |
whisper  |       ██╗     ███████╗██╗ ██████╗
whisper  |       ██║     ██╔════╝██║██╔═══██╗
whisper  |       ██║     ███████╗██║██║   ██║
whisper  |       ██║     ╚════██║██║██║   ██║
whisper  |       ███████╗███████║██║╚██████╔╝
whisper  |       ╚══════╝╚══════╝╚═╝ ╚═════╝
whisper  |
whisper  |    Brought to you by linuxserver.io
whisper  | ───────────────────────────────────────
whisper  |
whisper  | To support LSIO projects visit:
whisper  | https://www.linuxserver.io/donate/
whisper  |
whisper  | ───────────────────────────────────────
whisper  | GID/UID
whisper  | ───────────────────────────────────────
whisper  |
whisper  | User UID:    1000
whisper  | User GID:    1000
whisper  | ───────────────────────────────────────
whisper  | Linuxserver.io version: 2.0.0-pkg-b2138dcc-dev-550713281c67d3de93e9724577979e887a40ccdc-pr-23
whisper  | Build-date: 2024-10-29T09:22:43+00:00
whisper  | ───────────────────────────────────────
whisper  |
whisper  | [custom-init] No custom files found, skipping...
whisper  | INFO:__main__:Ready
whisper  | Connection to localhost (127.0.0.1) 10300 port [tcp/*] succeeded!
whisper  | [ls.io-init] done.
whisper  | INFO:faster_whisper:Processing audio with duration 00:02.150
whisper  | INFO:wyoming_faster_whisper.handler: Jaki mamy dziś dzień?
```

**Comment by MorganMLGman** [comment]
`ghcr.io/linuxserver/lspipepr-faster-whisper:gpu-2.0.0-pkg-b2138dcc-dev-f3a49ddf041bf5ebabd973b04027986e0cc3c451-pr-23`

Also works

```
whisper  | [migrations] started
whisper  | [migrations] no migrations found
whisper  | ───────────────────────────────────────
whisper  |
whisper  |       ██╗     ███████╗██╗ ██████╗
whisper  |       ██║     ██╔════╝██║██╔═══██╗
whisper  |       ██║     ███████╗██║██║   ██║
whisper  |       ██║     ╚════██║██║██║   ██║
whisper  |       ███████╗███████║██║╚██████╔╝
whisper  |       ╚══════╝╚══════╝╚═╝ ╚═════╝
whisper  |
whisper  |    Brought to you by linuxserver.io
whisper  | ───────────────────────────────────────
whisper  |
whisper  | To support LSIO projects visit:
whisper  | https://www.linuxserver.io/donate/
whisper  |
whisper  | ───────────────────────────────────────
whisper  | GID/UID
whisper  | ───────────────────────────────────────
whisper  |
whisper  | User UID:    1000
whisper  | User GID:    1000
whisper  | ───────────────────────────────────────
whisper  | Linuxserver.io version: 2.0.0-pkg-b2138dcc-dev-f3a49ddf041bf5ebabd973b04027986e0cc3c451-pr-23
whisper  | Build-date: 2024-10-29T09:28:25+00:00
whisper  | ───────────────────────────────────────
whisper  |
whisper  | [custom-init] No custom files found, skipping...
whisper  | INFO:__main__:Ready
whisper  | Connection to localhost (127.0.0.1) 10300 port [tcp/*] succeeded!
whisper  | [ls.io-init] done.
whisper  | INFO:faster_whisper:Processing audio with duration 00:02.110
whisper  | INFO:wyoming_faster_whisper.handler: Jak nam jeździ?
whisper  | INFO:faster_whisper:Processing audio with duration 00:02.660
whisper  | INFO:wyoming_faster_whisper.handler: Jaki mamy dziś dzień?
```

**Comment by thespad** [comment]
Thanks for testing, I'll get the PR reviewed and merged shortly.

**Comment by thespad** [comment]
Fixed via #23
